src/coreclr/jit/instr.h

```diff
@@ -146,8 +146,7 @@ enum insFlags : uint64_t
     // Avx
     INS_Flags_IsDstDstSrcAVXInstruction = 1ULL << 26,
     INS_Flags_IsDstSrcSrcAVXInstruction = 1ULL << 27,
-    INS_Flags_IsMskSrcSrcEvexInstruction = 1ULL << 28,
```

Comment:
As covered in another thread; `IsDstDstSrc` and `IsDstSrcSrc` were named based on an older need when the JIT was not truly VEX aware and didn't support tracking the destination and first input separately. Back then, these were basically used to determine which operand got duplicated to the `VEX.vvvv` bits.

In today's JIT where we're properly VEX aware they end up being non-ideal for names. `IsMskSrcSrc` was being used for an instruction where the destination was a `kmask` register, but that is largely an unimportant detail and was actually causing the operands to be incorrectly encoded in a few scenarios.

For now, I've removed it and made it be `IsDstDstSrc` which allows for the encodings to be correct.

Longer term we really want to replace these with something that better details what register is encoded where. This could be something like a `EncodeReg1InVex_vvvv` flag. It could also be expanded to also detail information such as `EncodeReg1InModRM_reg` or `EncodeReg3InModRM_rm`, etc.